apps/aecore/src/aec_genesis_block_settings.erl

```diff
@@ -0,0 +1,25 @@
+-module(aec_genesis_block_settings).
+
+-export([dir/0,
+         preset_accounts/0]).
+
+dir() ->
+    case application:get_env(aecore, genesis_settings_dir) of
```

Comment:
The genesis block shall not be configurable. Please correct me if I am wrong.

Reply:
The idea is having a couple of incompatible distinct networks (for example one for uat and one for the public nodes). This is how we'd guarantee that someone will not connect our distinct networks and make them peers.

Reply:
I see. So keep it.